schedule/yast/iscsi_ibft.yaml

```diff
@@ -16,7 +16,7 @@ schedule:
   system_role:
     - installation/system_role/accept_selected_role_SLES_with_GNOME
   guided_partitioning:
-    - installation/partitioning/select_guided_setup
+    - installation/partitioning
```

Comment:
we don't want to use needle-based modules, please keep using  `select_guided_setup` and  fix the setting via MR if it is this test the one that was doing it. We should change in the future the need for a setting.

Reply:
VR now successful with libyui based guided partitioning.